v1.2 made edits for 1.2 release (#130)
mulesoft/docs-service-mesh: modules/ROOT/pages/provision-adapter-configure-service-mesh-CRD.adoc

After you provision the adapter, you must set the `istio-injection` label on the namespace and redeploy all your existing applications in that namespace to ensure that the Envoy sidecar is injected within each pod in the Kubernetes Cluster.

> If you have installed Anypoint Service Mesh with Red Hat OpenShift, you must set the `istio-cni` network-attachment-definition on the namespace where you are running the applications.

Reviewer: ```suggestion
If you have installed Anypoint Service Mesh with Red Hat OpenShift, you must set the `istio-cni` network-attachment definition in your application namespace.
```
```suggestion
If you have installed Anypoint Service Mesh with Red Hat OpenShift, you must set the `istio-cni` network-attachment-definition on the namespace where you are running the applications.
```

Reviewer 2: Hi @kbostrom, the type of resource the user need to create is `network-attachment-definition` with the name `istio-cni`, not `network-attachment`

Author: Left as is. Made no changes.

`asmctl adapter list`
. Set the label for Istio in the namespace:
`kubectl label ns <namespace> istio-injection=enabled`

> . If have installed Anypoint Service Mesh with Red Hat OpenShift, run the following command on the namespace where your applications are running :

Reviewer: ```suggestion
. If have installed Anypoint Service Mesh with Red Hat OpenShift, run the following command in your application namespace:
```
```suggestion
. If have installed Anypoint Service Mesh with Red Hat OpenShift, run the following command on the namespace where your applications are running :
```

Author: Done! Thanks!

+
[source,text,linenums]
----
kubectl -n <namespace> apply -f - <<EOF

apiVersion: "k8s.cni.cncf.io/v1"
kind: NetworkAttachmentDefinition
metadata:
  name: istio-cni
EOF
----
+

> This command creates the resource to enable the Istio cni plugin in the application namespace.

Reviewer: ```suggestion
This command creates the resources that enable the Istio CNI plugin in the namespace.
```
```suggestion
This command creates the resource to enable the Istio cni plugin in the application namespace.
```

Author: Done! Thanks!

> . Redeploy all your existing applications in the namespace.

Reviewer: ```suggestion
. Redeploy all your existing namespace applications.
```
```suggestion
. Redeploy all your existing applications in the namespace.
```

Author: Left as is. Thanks!
